src/main/java/org/folio/rest/impl/GetOrdersHelper.java

```diff
@@ -0,0 +1,106 @@
+package org.folio.rest.impl;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionException;
+
+import javax.ws.rs.core.Response;
+
+import org.apache.log4j.Logger;
+import org.folio.orders.rest.exceptions.HttpException;
+import org.folio.orders.utils.HelperUtils;
+import org.folio.rest.jaxrs.model.CompositePurchaseOrders;
+import org.folio.rest.jaxrs.resource.OrdersResource.GetOrdersResponse;
+import org.folio.rest.tools.client.interfaces.HttpClientInterface;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Context;
+import io.vertx.core.Future;
+import io.vertx.core.Handler;
+import io.vertx.core.json.JsonObject;
+import me.escoffier.vertx.completablefuture.VertxCompletableFuture;
+
+public class GetOrdersHelper {
+
+  private static final Logger logger = Logger.getLogger(GetOrdersHelper.class);
+
+  public static final String MOCK_DATA_PATH = "src/test/resources/mockdata/getOrders.json";
+
+  private final HttpClientInterface httpClient;
+  private final Context ctx;
+  private final Handler<AsyncResult<javax.ws.rs.core.Response>> asyncResultHandler;
+  private final Map<String, String> okapiHeaders;
+
+  public GetOrdersHelper(HttpClientInterface httpClient, Map<String, String> okapiHeaders,
+      Handler<AsyncResult<Response>> asyncResultHandler, Context ctx) {
+    this.httpClient = httpClient;
+    this.okapiHeaders = okapiHeaders;
+    this.ctx = ctx;
+    this.asyncResultHandler = asyncResultHandler;
+  }
+
+  public CompletableFuture<CompositePurchaseOrders> getOrders(String query, int offset, int limit, String lang) {
+    CompletableFuture<CompositePurchaseOrders> future = new VertxCompletableFuture<>(ctx);
+
+    //TODO replace this with a call to mod-orders-storage
+    getMockOrders(query, offset, limit, lang)
```

Comment:
Is there a plan to keep this code to return mock orders in here when implementing a call to mod-orders-storage? If so then will need a way to easily get either (mock or real) objects.  I can see the benefits of having the functionality to return mock data for testing.

Reply:
I don't think so. eventually this code would be replaced by the "real" implementation.  Instead of returning mock data, we'd populate the system with reference or test data, depending on the situation.